### ISSUE TITLE
Disable automatic scroll down in serial monitor

### ISSUE THREAD

**Issue by lonas123** [platformio/platformio-atom-ide-terminal#28]
Please add option for disable automatic scroll down in serial monitor and scroll on the right to easily review serial monitor output
##

**Comment by proddy** [comment]
+1 for this feature. Its one of the things I liked with the Arduino IDE

**Comment by artem469** [comment]
+1 too

**Comment by Peouse** [comment]
+1 , that would be nice for users without the debugging mode.

**Comment by salami738** [comment]
+1 too

**Comment by horeich** [comment]
+1 would very handy imo

**Comment by krisha** [comment]
hm, +1
and maybe pls also change default baudrate to 115200? (maybe not related to this repository)

**Comment by wacher74** [comment]
+1 It would be handy sometimes

**Comment by greenovationtechnolads** [comment]
Please add this feature. It is a really handy feature. Currently i have to use arduino's serial monitor and everytime i upload code i have close it and again restart it.

**Comment by eueduardo3** [comment]
com certeza isso seria util

**Comment by UziTech** [comment]
This package is no longer maintained. Please uninstall platformio-ide-terminal and install either [terminus](https://atom.io/packages/terminus) or [x-terminal](https://atom.io/packages/x-terminal)

**Comment by moedris** [comment]
+1

**Comment by soundprayer** [comment]
+1

**Comment by gecko9** [comment]
if no longer maintained then pls give instructions how to install another terminal into PlatformIO otherwise I vote to optionally disable auto-scroll using a toggle button

**Comment by russbrookes** [comment]
I agree - ability to turn off auto scroll is needed. Often times by the time you press ctrl-c to stop the scrolling you've lost the lines at the top you wanted to see. And sometimes you don't want to stop the monitor with ctrl-c, but you do want to scroll back up and see the first few lines (without being hampered by new lines being written to the monitor), and then scroll back to the bottom, and then let auto scrolling occur again.
